### DISCUSSION TITLE
question regarding memory

### DISCUSSION THREAD

**Discussion by Kartikj112** [metagenome-atlas/atlas#732] [Q&A]
Hi atlas community  

Pardon my naivety, I'm a noob at metagenomics and informatics in general. I have done some bacterial genome assembly work in past.

My question is regarding the required memory for a small metagenomics project to run atlas pipeline.

The documentation says: 
> For real metagenomic data atlas should be run on a _linux_ system, with enough memory (min ~50GB but assembly usually requires 250GB).

I have been trying to decide the sequencing depth needed and the expected raw reads file size. The sequencing companies have been quoting for a raw reads file size of 4-6 GB per sample. The literature mentions file size ranging from ten to hundreds of GBs. I understand that the file size and depth depend on the type of sample, so there is no one size fits all and depends on the question being addressed. Having said that,

**My question is if the size of my single sample raw read file is 4-6 GB, should I process it using a 64-128 GB RAM PC?**
Any other advice would be helpful.

Regards

Kartik

**Comment by SilasK** [comment]
The file size of a fastq.gz file is  most of the time not so important and depends on the diversity of your metagenome. 
I'm not sure if the company was not talking about Gbp = bases )

Anyway, the assembly memory for one sample could fit on 128Gb RAM.


Take also into consideration that atlas for some annotations use databases that need 100GB + disk storage .
I sugest you to start using atlas run assembly first ..

**Reply by Kartikj112** [reply]
Thankyou. This was really helpful :D
